etc/kayobe/dnf.yml

```diff
@@ -90,6 +90,8 @@ dnf_custom_repos_centos:
     file: CentOS-Stream-BaseOS
     gpgkey: file:///etc/pki/rpm-gpg/RPM-GPG-KEY-centosofficial
     gpgcheck: yes
+    # Exclude buggy iptables: https://bugzilla.redhat.com/show_bug.cgi?id=2236501
+    exclude: "iptables-1.8.5*"
```

Comment:
```suggestion
    exclude: "iptables-1.8.5* iptables-libs-1.8.5* iptables-ebtables-1.8.5* iptables-services-1.8.5*"
```
In testing, not pinning these additional packages broke: `sudo dnf update -y` without ``--nobest``:

```
[stack@osnode3d10 ~]$ sudo dnf update -y                                                                                                                                      CentOS Stream 8 - BaseOS                                                                                                                      182 kB/s | 3.9 kB     00:00
Error:                                                                   
 Problem 1: package iptables-services-1.8.5-8.el8.x86_64 requires iptables = 1.8.5-8.el8, but none of the providers can be installed
  - cannot install the best update candidate for package iptables-services-1.8.4-23.el8.x86_64                                                                               
  - package iptables-1.8.5-8.el8.i686 is filtered out by exclude filtering  
  - package iptables-1.8.5-8.el8.x86_64 is filtered out by exclude filtering
 Problem 2: package iptables-ebtables-1.8.5-8.el8.x86_64 requires iptables = 1.8.5-8.el8, but none of the providers can be installed                                         
  - cannot install the best update candidate for package iptables-ebtables-1.8.4-23.el8.x86_64                                                                               
  - package iptables-1.8.5-8.el8.i686 is filtered out by exclude filtering                                                                                                   
  - package iptables-1.8.5-8.el8.x86_64 is filtered out by exclude filtering                                                                                                 
 Problem 3: package centos-release-kmods-rebuild-2-1.el8.noarch requires centos-release-kmods = 2-1.el8, but none of the providers can be installed                             - cannot install both centos-release-kmods-8-3.el8s.noarch and centos-release-kmods-2-1.el8.noarch
  - cannot install the best update candidate for package centos-release-kmods-rebuild-2-1.el8.noarch
  - cannot install the best update candidate for package centos-release-kmods-2-1.el8.noarch
  ```